nemo/collections/nlp/models/language_modeling/megatron/bert/bert_spec.py

```diff
@@ -0,0 +1,106 @@
+from megatron.core.fusions.fused_bias_dropout import get_bias_dropout_add
+from megatron.core.fusions.fused_layer_norm import FusedLayerNorm
+from megatron.core.tensor_parallel.layers import ColumnParallelLinear, RowParallelLinear
+from megatron.core.transformer.attention import SelfAttention, SelfAttentionSubmodules
+from megatron.core.transformer.custom_layers.transformer_engine import (
+    TEColumnParallelLinear,
+    TEDotProductAttention,
+    TELayerNormColumnParallelLinear,
+    TENorm,
+    TERowParallelLinear,
+)
+from megatron.core.transformer.dot_product_attention import DotProductAttention
+from megatron.core.transformer.enums import AttnMaskType
+from megatron.core.transformer.mlp import MLP, MLPSubmodules
+from megatron.core.transformer.spec_utils import ModuleSpec
+from megatron.core.transformer.transformer_layer import TransformerLayer, TransformerLayerSubmodules
+
+from nemo.collections.nlp.models.language_modeling.megatron.bert.bert_model import (
+    TransformerLayerPostLNSupport,
+    TransformerLayerSubmodulesPostLNSupport,
+)
+
+# Use this spec to use lower level Transformer Engine modules (required for fp8 training)
+bert_layer_with_transformer_engine_spec = ModuleSpec(
```

Comment:
Why are we adding this spec ? Itis the same as the mcore bert_layer_with_Transformer_engine_Spec. You can just import it here . Same for the local spec asl= well without post layer norm